PocoDoc/Makefile

```diff
@@ -10,6 +10,10 @@ objects = DocWriter PocoDoc
 
 target         = PocoDoc
 target_version = 1
-target_libs    = PocoCppParser PocoFoundation PocoXML PocoUtil
+target_libs    = PocoCppParser PocoDataSQLite PocoData PocoFoundation PocoXML PocoUtil
+
+ifdef POCO_ENABLE_SQLITE_FTS5
+CXXFLAGS += -DPOCO_ENABLE_SQLITE_FTS5
```

Comment:
Same here to enable full text search.

Reply:
Fixed in ef13d0fab675b72f6db2bc185d1ecab1a2494b31